docs/installation.rst

```diff
@@ -204,7 +204,7 @@ Commit `a2fe711ffa3f` (an older version of the `master` branch) was tested to wo
 
     module load anaconda3/2020.11 cudatoolkit/11.1 cudnn/cuda-11.1/8.0.4
 
-    conda create --name desc-env python=3.8
```

Comment:
was this tested to work? I guess it should not really matter but the power architectures were tricky to work with install-wise

Reply:
i had 3.9 installed when i was running stuff on traverse